AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestRunnerService.kt

```diff
@@ -91,6 +92,11 @@ interface TestRunnerService {
      */
     suspend fun getTestMatrixResults(testMatrix: TestMatrix): List<TestRunResult>?
 
+    /**
+     * Downloads the file for the given [inputFile] GcsPath to the outputFile.
+     */
+    suspend fun downLoadFileFromGcloud(outputFile: File, inputFile: GcsPath)
```

Comment:
we should return a stream instead of asking an output file so that we can use it w/o saving it to somewhere on the androidx.dev side.

you can check how we read the test result xml files.